library/pcs_cluster.py

```diff
@@ -141,13 +149,12 @@ def run_module():
                 re_nodes_list = nodes.findall(corosync_conf.read())
                 re_node_list_set = set()
                 if len(re_nodes_list) > 0:
-                    n_name = re.compile(r"ring0_addr\s*:\s*([\w.-]+)\s*", re.M)
+                    n_name = re.compile(r"ring[0-9]+_addr\s*:\s*([\w.-]+)\s*", re.M)
```

Comment:
```suggestion
                    n_name = re.compile(r"ring[01]+_addr\s*:\s*([\w.-]+)\s*", re.M)
```
I have not seen more than ring1_addr so I would stay matching only `ring0_addr` and `ring1_addr`

Reply:
I wouldn't fix this on 0 and/or 1, as there is technically no limit to the number of redundant rings.

Reply:
While `man corosync.conf` mentions that this is `ringnumber` where `number` should be unique I have not encountered yet clusters with anything above `ring1_addr`. Do you have setup where anything beyond `ring0_addr` and `ring1_addr` is needed? 
If yes, could you please share link on how to setup such cluster ideally with `pcs`? 

Trying just adding another address seems to make corosync unhappy on CentOS 7.6 with error below that seems to point to limit of 2 addresses.
~~~
nodelist {
    node {
        ring0_addr: node1-eth0.example.com
        ring1_addr: node1-eth1.example.com
        ring2_addr: node1-eth2.example.com
        nodeid: 1
    }

    node {
        ring0_addr: node2-eth0.example.com
        ring1_addr: node2-eth1.example.com
        ring2_addr: node2-eth2.example.com
        nodeid: 2
    }
}

[MAIN  ] parse error in config: interface ring number 2 is bigger than allowed maximum 1
[MAIN  ] Corosync Cluster Engine exiting with status 8 at main.c:1373.

pacemaker-1.1.19-8.el7_6.1.x86_64
~~~

Reply:
moved to #22